erica_app/erica/pyeric/eric.py

```diff
@@ -184,21 +184,15 @@ def close_cert_handle(self, cert_handle):
 
     def get_cert_properties(self):
         fun_get_cert_properties = self.eric.EricMtHoleZertifikatEigenschaften
-        fun_get_cert_properties.argtypes = [c_void_p, c_int, c_char_p, c_void_p]
-        fun_get_cert_properties.restype = c_int
-
-        cert_handle = None
-        buffer = None
+        argtypes = [c_void_p, c_int, c_char_p, c_void_p]
+        restype = c_int
```

Comment:
Yes, I was thinking about setting them directly, too. In the previous approach we used the function directly afterwards. Because the function is now one parameter I wanted to make it explicit that you have to set the argtypes and restype beforehand. 

That said, as you also like that approach more. I'll change it and just add a docstring to the function to explain how to use it.